ATeam-FantasyGame/Location.cpp

```diff
@@ -0,0 +1,23 @@
+#include "Location.h"
+
+//Location class
+
+//Constructor
+Location::Location(std::string name, std::string description)
+{
```

Comment:
this may not do what you think it does.
suggest changing the argument names to be different from the instance variable names.
If you really want to keep the names identical, look up the usage of "this." to reference the instance variables.